Gemfile

```diff
@@ -13,7 +14,7 @@ gem 'omniauth-slack'
 gem 'cobot_client'
 gem 'sentry-raven'
 gem 'lograge'
-gem 'cobot_assets', '~> 0.0.9'
+gem 'cobot_assets', '~> 17.9.0'
```

Comment:
We moved to a higher version (18.4.3.) in the meantime, can be found here: https://github.com/upstream/cobot_assets/releases

Reply:
Fixed!